rust/examples/mcap/src/main.rs

```diff
@@ -41,13 +41,20 @@ impl From<CompressionArg> for Option<McapCompression> {
     }
 }
 
-#[derive(Debug, foxglove::Loggable)]
+#[derive(Debug, foxglove::Encode)]
 struct Message {
     msg: String,
     count: u32,
 }
 
+#[derive(Debug, serde::Serialize, schemars::JsonSchema)]
+struct JsonMessage {
+    msg: String,
+    count: u32,
+}
+
 static MSG_CHANNEL: LazyChannel<Message> = LazyChannel::new("/msg");
+static JSON_CHANNEL: LazyChannel<JsonMessage> = LazyChannel::new("/json");
```

Comment:
I am so/so on this. On one hand it is cool to see you can write binary and JSON to a file - on the other it makes the example more "noisy"